packages/analytics-js/src/components/core/Analytics.ts

```diff
@@ -91,6 +91,7 @@ class Analytics implements IAnalytics {
     this.preloadBuffer = new BufferQueue();
     this.initialized = false;
     this.errorHandler = defaultErrorHandler;
+    this.errorHandler.attachErrorListeners();
```

Comment:
we should use that in the RudderAnalytics class (app folder) not this one. This is created only after load method is invoked.

Reply:
Addressed